### DISCUSSION TITLE
Open multiple files besides just a folder or a single file

### DISCUSSION THREAD

**Discussion by IArnaut2** [AssetRipper/AssetRipper#912] [Ideas]
Can you make it to be able to select multiple files besides just a folder or a single file?

![image](https://github.com/AssetRipper/AssetRipper/assets/57326929/e3de7c4e-b434-4c04-bd62-968d80073ee2)

**Comment by ds5678** [comment]
You're not the first one to mention this, but I can't remember where the previous conversation is. Anyway, it used to work the way you describe, but then Avalonia had some changes that broke it.
